AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -189,7 +188,7 @@ define([
          * @param {Object|null} relatedImage
          * @param {Object} record
```

Comment:
Please remove the `record` parameter declaration

Reply:
Good eye! done.